modin/pandas/dataframe.py

```diff
@@ -861,15 +858,15 @@ def _update_var_dicts_in_kwargs(self, expr, **kwargs):
             if kwargs.get("global_dict"):
                 global_dict.update(kwargs.get("global_dict"))
             kwargs["global_dict"] = global_dict
-        return kwargs
+        return
```

Comment:
just delete the statement, a lone `return` in the end of function is not necessary :)